boards/riscv/longan_nano/Kconfig.defconfig

```diff
@@ -8,4 +8,20 @@ config BOARD
 	default "longan_nano" if BOARD_LONGAN_NANO
 	default "longan_nano_lite" if BOARD_LONGAN_NANO_LITE
 
+# This definition automatically resolves the dependency to GPIO
+# from spi1/cs-gpios defined in the DTS file.
+config GPIO
+	bool
+	default y if $(dt_nodelabel_enabled,spi0) && $(dt_nodelabel_has_prop,spi0,cs-gpios)
```

Comment:
why is this required? please just add `CONFIG_GPIO=y` to board defconfig if not already there (gpio is considered a "basic" peripheral so it can be enabled by default).